app/models/page.rb

```diff
@@ -25,6 +25,30 @@ class Page < ApplicationRecord
 
   attribute :answer_settings, DataStructType.new
 
+  def self.create_and_update_form!(form_id:,
+                                   question_text:,
+                                   hint_text:,
+                                   is_optional:,
+                                   is_repeatable:,
+                                   answer_settings:,
+                                   page_heading:,
+                                   guidance_markdown:,
+                                   answer_type:)
+    page = Page.new(
+      form_id:,
+      question_text:,
+      hint_text:,
+      is_optional:,
+      is_repeatable:,
+      answer_settings:,
+      page_heading:,
+      guidance_markdown:,
+      answer_type:,
+    )
```

Comment:
Rather than listing all the attributes and creating extra work for ourself, we can use the Ruby argument forwarding syntax [[1]]

```suggestion
  def self.create_and_update_form!(...)
    page = Page.new(...)
```

[1]: https://docs.ruby-lang.org/en/3.4/syntax/methods_rdoc.html#label-Argument+Forwarding